libraries/botbuilder-ai/tests/luisRecognizer.test.js

```diff
@@ -276,7 +276,13 @@ describe('LuisRecognizer', () => {
         assert(res.intents);
         assert(res.intents.Delivery);
         assert(res.intents.Delivery.score > 0 && res.intents.Delivery.score <= 1);
+
         assert(LuisRecognizer.topIntent(res) === 'Delivery');
+
+        const sortedIntents = LuisRecognizer.sortedIntents(res);
+        assert(sortedIntents[0].intent) === 'Delivery';
+        assert(sortedIntents[sortedIntents.length - 1].intent) === 'EntityTests';
```

Comment:
Does the "intent2" code need a distinct unit test or is it already covered in existing tests?